packages/shared/src/locales/en.json

```
@@ -956,6 +956,9 @@
             "action": "Sign",
             "success": "Signed message"
         },
+        "siwe": {
+            "title": "Sign-in-with-Ethereum"
+        },
```

Follow the standard's title: 

```suggestion
        "siwe": {
            "title": "Sign-In with Ethereum"
        },
```